### PR TITLE
chore: update release note template (V23.3) [skip ci]

### PR DESCRIPTION
**Pull request by ZheSun88**

### CODE REVIEW

File: scripts/generator/templates/template-release-notes-maintenance.md [hunk 1]
```diff
@@ -16,12 +10,18 @@ Vaadin {{platform}}
...
-- Collaboration Engine ([{{vaadin.vaadin-collaboration-engine.javaVersion}}](https://github.com/vaadin/collaboration-engine/releases/tag/{{vaadin.vaadin-collaboration-engine.javaVersion}}))
-- Designer ([Release notes](https://github.com/vaadin/designer/blob/master/RELEASE-NOTES.md))
 - TestBench ([{{vaadin.vaadin-testbench.javaVersion}}](https://github.com/vaadin/testbench/releases/tag/{{vaadin.vaadin-testbench.javaVersion}}))
 - Classic Components([{{vaadin.vaadin-classic-components.javaVersion}}](https://github.com/vaadin/classic-components/releases/tag/{{vaadin.vaadin-classic-components.javaVersion}}))
 - Multiplatform Runtime (MPR) ([{{core.mpr-v8.javaVersion}}](https://github.com/vaadin/multiplatform-runtime/releases/tag/{{core.mpr-v8.javaVersion}}))
 - Router ([{{core.vaadin-router.jsVersion}}](https://github.com/vaadin/vaadin-router/releases/tag/v{{core.vaadin-router.jsVersion}}))
+- Vaadin Kits
+  - Azure Kit ([{{kits.azure-kit.version}}](https://vaadin.com/docs/latest/tools/azure))
+  - Collaboration Engine ([{{kits.vaadin-collaboration-engine.javaVersion}}](https://github.com/vaadin/collaboration-engine/releases/tag/{{vaadin.vaadin-collaboration-engine.javaVersion}}))
```
Comment (manolo) [inline scripts/generator/templates/template-release-notes-maintenance.md:19]: Change this to Collaboration Kit (aka Collaboration Engine)

File: scripts/generator/templates/template-release-notes-maintenance.md [hunk 2]
```diff
@@ -16,12 +10,18 @@ Vaadin {{platform}}
...
   - Flow Components ([{{platform}}](https://github.com/vaadin/flow-components/releases/tag/{{platform}}))
-- Collaboration Engine ([{{vaadin.vaadin-collaboration-engine.javaVersion}}](https://github.com/vaadin/collaboration-engine/releases/tag/{{vaadin.vaadin-collaboration-engine.javaVersion}}))
-- Designer ([Release notes](https://github.com/vaadin/designer/blob/master/RELEASE-NOTES.md))
 - TestBench ([{{vaadin.vaadin-testbench.javaVersion}}](https://github.com/vaadin/testbench/releases/tag/{{vaadin.vaadin-testbench.javaVersion}}))
 - Classic Components([{{vaadin.vaadin-classic-components.javaVersion}}](https://github.com/vaadin/classic-components/releases/tag/{{vaadin.vaadin-classic-components.javaVersion}}))
 - Multiplatform Runtime (MPR) ([{{core.mpr-v8.javaVersion}}](https://github.com/vaadin/multiplatform-runtime/releases/tag/{{core.mpr-v8.javaVersion}}))
 - Router ([{{core.vaadin-router.jsVersion}}](https://github.com/vaadin/vaadin-router/releases/tag/v{{core.vaadin-router.jsVersion}}))
+- Vaadin Kits
+  - Azure Kit ([{{kits.azure-kit.version}}](https://vaadin.com/docs/latest/tools/azure))
```
Comment (manolo) [inline scripts/generator/templates/template-release-notes-maintenance.md:18]: I would put Collaboration in the first position, even though it's break the alphabetical order here

File: scripts/generator/templates/template-release-notes.md [hunk 3]
```diff
@@ -2,41 +2,67 @@ Vaadin {{platform}}
...
-`*` experimental
+- Stable **Spreadsheet** component for flow
+- New component: **TabSheet**
+- New component: **Tooltip** 
+- Chain validation in inputs: 1. web-component, 2. Flow component, 3. Binder.
+- **DatePicker**: 2-digit year parsing
+- Shift-click for **Grid** multi-sort
+
+### Vaadin Kits
+- Collaboration Engine
```
Comment (manolo) [inline scripts/generator/templates/template-release-notes.md:36]: Rename to Collaboration Kit
Include changes in CE from 5.2.0 to 5.3.0

**Review by manolo — CHANGES_REQUESTED** [review]

File: scripts/generator/templates/template-release-notes.md (hunk 3 above)
Comment (ZheSun88) [reply scripts/generator/templates/template-release-notes.md]: done

File: scripts/generator/templates/template-release-notes-maintenance.md (hunk 2 above)
Comment (ZheSun88) [reply scripts/generator/templates/template-release-notes-maintenance.md:19]: updated.

File: scripts/generator/templates/template-release-notes-maintenance.md (hunk 1 above)
Comment (ZheSun88) [reply scripts/generator/templates/template-release-notes-maintenance.md]: updated.

**Review by manolo — APPROVED** [review]

**Pull request merged**